tests/internal/py/logic.py

```diff
@@ -0,0 +1,19 @@
+# Copyright Metatype OÜ, licensed under the Elastic License 2.0.
+# SPDX-License-Identifier: Elastic-2.0
+
+
+def remote_sum(inp: dict, gql) -> float:
+    data = gql(
```

Comment:
Try to emulate the existing custom function impls by passing in a ctx object instead of the `gql` function directly